lib/reporter.js

```diff
@@ -86,16 +86,43 @@ module.exports = class Reporter {
     const github = await this.getGithub();
     const login = user.login.toLowerCase();
     const org = this.installation.account.login;
+
     // review:none returns all PRs without a review
     // sadly there is a bug in github that if you commit after
     // someone requested changes, the pr is still reviewed
     // so it will no longer show up whenever someone already reviewed it ever
-    const query = `is:pr is:open review:none review-requested:${login} org:${org}`;
-    const request = github.search.issues({ q: query, per_page: 100 });
-    const pullRequests = await github.paginate(request, response => response.data.items
-      .filter(pr => this.isStale(pr)));
-
-    return _.sortBy(pullRequests, pr => pr.created_at);
+    const toReview = await github.paginate(
+      github.search.issues({
+        q: `review:none is:pr is:open review-requested:${login} org:${org}`,
+        per_page: 100,
+      }),
+      response => response.data.items
+        .filter(pr => this.isStale(pr)));
+
+    // This query returns all PRs that already have been reviewed
+    // and need to be completed by the assignee
+    // we need to make two quries with
+    // review:changes_requested and review:approved
+    // since github is not supporting OR
+    const changesRequested = await github.paginate(
+      github.search.issues({
+        q: `review:changes_requested is:pr is:open assignee:${login} org:${org}`,
+        per_page: 100,
+      }),
+      response => response.data.items);
+    const approved = await github.paginate(
+      github.search.issues({
+        q: `review:approved is:pr is:open assignee:${login} org:${org}`,
+        per_page: 100,
+      }),
+      response => response.data.items);
+
+    const toComplete = _.concat(changesRequested, approved);
```

Comment:
How about just using one query like this: `-review:none is:pr is:open assignee:${login} org:${org}`?